### DISCUSSION TITLE
제페토 패키지 빌드를 자동화할 방법이 있을까요?

### DISCUSSION THREAD

**Discussion by hyunjoon-park-metaspace** [naverz/zepeto-studio-kor#1007] [5. Q&A]
안녕하세요, 불철주야 고생이 많으십니다.

빌드 과정을 자동화하고 싶습니다. 유니티 에디터는 [`BuildPipeline.BuildPlayer`](https://docs.unity3d.com/ScriptReference/BuildPipeline.BuildPlayer.html)을 이용하면 CLI를 통해 빌드를 자동으로 수행할 수 있는데요, 제페토도 이런 기능과 비슷한 기능을 지원할까요?

감사합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 건의 주신 내용에 공감하며
추후에라도 제페토 패키지 빌드를 자동화하는 방법을 지원할 수 있도록 검토해 보겠습니다.
감사합니다.
